OpenRA.Platforms.Default/Sdl2PlatformWindow.cs

```diff
@@ -92,11 +93,13 @@ public Size SurfaceSize
 		[DllImport("user32.dll")]
 		static extern bool SetProcessDPIAware();
 
-		public Sdl2PlatformWindow(Size requestEffectiveWindowSize, WindowMode windowMode, float scaleModifier, int batchSize)
+		public Sdl2PlatformWindow(Size requestEffectiveWindowSize, WindowMode windowMode, float scaleModifier, int batchSize, int videoDisplay)
 		{
 			// Lock the Window/Surface properties until initialization is complete
 			lock (syncObject)
 			{
+				SDL.SDL_VideoInit(SDL.SDL_GetCurrentVideoDriver());
```

Comment:
Why is this needed? The `SDL_Init` line just below should already be doing this.

Reply:
It is not, thx.